### PR TITLE
fix: x% match labels do not align with project titles in sponsor cards

### PR DESCRIPTION
**Pull request by sk-tenba**
https://app.asana.com/0/1203842063837585/1204133675855752

### CODE REVIEW

**Comment by sk-tenba** [comment]
image

File: containers/PageHome/containers/ProjectCarousel/components/ProjectCard/index.module.scss [hunk 1]
```diff
@@ -96,7 +96,15 @@
...
-  left: 56px;
+
+  &.normal {
+    left: 56px;
+    top: 32px;
+  }
+  &.narrow {
+    left: 32px;
+    top: 16px;
+  }
```
Comment (sk-kitsune) [inline containers/PageHome/containers/ProjectCarousel/components/ProjectCard/index.module.scss:107]:
```
.normal & {...}
.narrow & {...}
```

File: containers/PageHome/containers/ProjectCarousel/components/ProjectCard/index.tsx [hunk 2]
```diff
@@ -92,7 +92,10 @@ export default function ProjectCard({
         <>
           {!value.match ? null : (
             <Typography.Div
-              className={styles.match}
+              className={cx(
+                styles.match,
+                padding === "normal" ? styles.normal : styles.narrow
```
Comment (sk-kitsune) [inline containers/PageHome/containers/ProjectCarousel/components/ProjectCard/index.tsx:97]: don't need

Comment (sk-kitsune) [reply containers/PageHome/containers/ProjectCarousel/components/ProjectCard/index.tsx:97]: btw, it's better to rename `styles.normal` to `styles.paddingNormal` and `styles.narrow` to `style.paddingNarrow`

**Review by sk-kitsune — APPROVED** [review]
✅ **Approved**: no blocking issues

**Pull request merged**